hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/operations/CustomCreateOperation.java

```diff
@@ -48,7 +48,13 @@ protected long cost(@NonNull final MessageFrame frame) {
     @Override
     protected @NonNull Address setupPendingCreation(@NonNull final MessageFrame frame) {
         final var updater = (ProxyWorldUpdater) frame.getWorldUpdater();
-        final var address = updater.setupInternalCreate(frame.getRecipientAddress());
+
+        final var origin = frame.getRecipientAddress();
+        final var originNonce = updater.getAccount(origin).getNonce();
+        // Decrement nonce by 1 to normalize the effect of transaction execution
+        final var address = Address.contractAddress(origin, originNonce - 1);
+
+        updater.setupInternalAliasedCreate(origin, address);
```

Comment:
So this also creates a PendingCreation.  Is this a different use case then the one created in TransactionProcessor.computeInvolvedParties -> updater.setupTopLevelLazyCreate?

Reply:
Yes, these result in different `createAccount()` behaviors, since the created account's Hedera-specific properties (e.g. auto-renew period) come from different sources.